codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/protocols/eventstream/ClientEventStreamMarshallerGeneratorTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.client.smithy.protocols.eventstream
+
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.ArgumentsProvider
+import org.junit.jupiter.params.provider.ArgumentsSource
+import software.amazon.smithy.rust.codegen.client.smithy.ClientCodegenContext
+import software.amazon.smithy.rust.codegen.core.smithy.CodegenTarget
+import software.amazon.smithy.rust.codegen.core.smithy.RuntimeType
+import software.amazon.smithy.rust.codegen.core.smithy.protocols.Protocol
+import software.amazon.smithy.rust.codegen.core.smithy.protocols.serialize.EventStreamMarshallerGenerator
+import software.amazon.smithy.rust.codegen.core.testutil.EventStreamTestModels
+import software.amazon.smithy.rust.codegen.core.testutil.EventStreamTestTools
+import software.amazon.smithy.rust.codegen.core.testutil.EventStreamTestVariety
+import software.amazon.smithy.rust.codegen.core.testutil.TestEventStreamProject
+import software.amazon.smithy.rust.codegen.core.testutil.TestRuntimeConfig
+import java.util.stream.Stream
+
+class MarshallTestCasesProvider : ArgumentsProvider {
+    override fun provideArguments(context: ExtensionContext?): Stream<out Arguments> =
+        // Don't include awsQuery or ec2Query for now since marshall support for them is unimplemented
```

Comment:
Do we have an issue related to this? It'd be nice to convert this into a linked TODO

Reply:
I don't think there's actually anything to do here since those protocols don't support event streams. I will remove the "is unimplemented" piece to make it clearer.

From the [AWS Query docs](https://smithy.io/2.0/aws/protocols/aws-query-protocol.html):
> This protocol does not support any kind of streaming requests or responses, including event streams.

The [EC2 Query docs](https://smithy.io/2.0/aws/protocols/aws-ec2-query-protocol.html) don't mention event streams at all, while all the protocol specs that support event streams mention them.

Reply:
In fact, I'll just delete the test cases for them entirely.

Reply:
Fixed in 80ed7a9510c43a6cbb030c047694b326de53c15a.